src/items.cpp

```diff
@@ -111,7 +111,7 @@ FILELOADER_ERRORS Items::loadFromOtb(const std::string& file)
 		return ERROR_INVALID_FORMAT;
 	}
 
-	for(auto & itemNode : root.children) {
+	for (auto & itemNode : root.children) {
```

Comment:
While at it, remove that space before &

Reply:
@ranisalt i have had fixed it before but somehow i rebased and it gone away :rofl: